modules/local/picrust.nf

```diff
@@ -18,7 +18,10 @@ process PICRUST {
     path("*_descrip.tsv"), emit: pathways
     path "versions.yml"  , emit: versions
     path "*.args.txt"    , emit: args
-    path "${message}.txt"
+    path "${message}.txt", emit: message
```

Comment:
missing `versions.yml`

Reply:
versions.yml is there, line 19, but emit was missing, yes.